templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+ <ul>
+    <li>
+        <a href="http://127.0.0.1:8000/">Home page</a>
+    </li>
+     <li>
+         <a href="http://127.0.0.1:8000/">Manufacturers</a>
```

Comment:
The href attribute for the 'Manufacturers' link is hardcoded with a local address. Consider using a URL template tag to dynamically generate the URL.